### PR TITLE
Tag Sims.jl v0.2.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [tshort/Sims.jl](https://github.com/tshort/Sims.jl)
Release: [v0.2.0](https://github.com/tshort/Sims.jl/releases/tag/v0.2.0)
Travis: [Travis Build Status](https://travis-ci.org/tshort/Sims.jl/builds/186336423)
Diff: [vs v0.1.0](https://github.com/tshort/Sims.jl/compare/71645586c9c1f82c60ea415ed9fa90575d8b83f2...48f771f9331596d595164d65e36b96677b71d267)
`requires` vs v0.1.0: 
```diff
--- 0.1.0/requires
+++ 0.2.0/requires
@@ -1,3 +1,8 @@
-julia 0.3-
-Requires
-Sundials+julia 0.5
+Compat
+Documenter
+Ipopt
+JuMP
+Plots
+ReactiveBasics 
+Sundials
```
cc: @tshort

### CODE REVIEW

File: Sims/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,8 @@
+julia 0.5
+Compat
```
Comment (tkelman) [inline Sims/versions/0.2.0/requires]: It's best to declare a minimum version dependency that implements the features you need.

**Comment by tshort** [comment]
Good point. I don't see a NEWS file for Compat. Is there a way to track changes other than commit logs?

**Comment by tkelman** [comment]
blame on the readme - see the note there

**Comment by tshort** [comment]
Given that I bumped the Julia version, I don't need Compat anymore. I'll delete the tag, and redo it, and attobot should resubmit the PR.

**Comment by ViralBShah** [comment]
So should we close this one? I love the Sims.jl demo, and hope there will be more.

**Comment by tshort** [comment]
Yes to closing. That's one quirk with attobot. I can't close this myself.

**Comment by kmsquire** [comment]
If you'll be using the same tag, then this should be left open. As @simonbyrne noted elsewhere, attobot will do the right thing if the PR is open (the PR will be updated), but if it's closed, attobot will force push to the same branch, and GitHub won't allow the PR to be reopened. 

(If you're bumping the tag as well, ten this can be closed).

**Comment by tshort** [comment]
I bumped the tag, so this can be closed.

**Comment by kmsquire** [comment]
And I see now that you bumped the tag. This means that there won't be a v0.2.0 tag then?

If you want to reuse this PR and tag, you can delete both the tag and release through GitHub, and then recreate them with the correct commit.

**Comment by tshort** [comment]
Good to know for next time. I just deleted the 0.2.0 release.

**Comment by tkelman** [comment]
don't think github allows deleting tags through the site, only releases - if you delete a release it leaves the tag in place

**Pull request closed without merging**